src/main/java/me/botsko/prism/database/sql/SQLIdMapQuery.java

```diff
@@ -100,6 +123,12 @@ public void findAllIds(String material, Consumer<List<IntPair>> success) {
         findAllIds(material, success, SQLIdMapQuery::noop);
     }
 
+    /**
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [First sentence of Javadoc is missing an ending period.](https://app.codacy.com/gh/AddstarMC/Prism-Bukkit/pullRequest?prid=5438808)